apis/apis.json

```diff
@@ -200,16 +200,16 @@
 
   {
     "id": "Google.Cloud.Logging.Log4Net",
-    "version": "2.0.0",
+    "version": "2.1.0",
     "type": "other",
     "targetFrameworks": "netstandard1.5;net45",
     "testTargetFrameworks": "netcoreapp1.0;net452",
     "description": "Log4Net client library for the Google Stackdriver Logging API.",
     "tags": [ "Log4Net", "Logging", "Stackdriver" ],
     "dependencies": {
       "log4net": "2.0.8",
-      "Google.Cloud.Logging.V2": "",
-      "Google.Cloud.DevTools.Common": "",
+      "Google.Cloud.Logging.V2": "2.0.0",
```

Comment:
Is this explicit just as a precaution, or is this now required because the versions of log4net/logging are now different?